lib/tasks/avalon.rake

```diff
@@ -129,6 +129,21 @@ EOC
     `rake db:migrate`
     `rails generate active_annotations:install`
   end
+
+  desc "Index MasterFiles and subresources to take advantage of SpeedyAF"
+  task index_for_speed: :environment do
+    MasterFile.find_each do |mf|
+      $stderr.print "m["
+      mf.update_index;
+      mf.declared_attached_files.each_pair do |name, file|
+        $stderr.print name.to_s[0]
+        file.update_external_index if file.respond_to?(:update_external_index)
```

Comment:
Nevermind...found it: https://github.com/projecthydra-labs/speedy_af/blob/master/lib/speedy_af/indexed_content.rb#L6-L8